### ISSUE TITLE
Linktext on an panel is not readable on focus

### ISSUE THREAD

**Issue by Febakke** [digdir/designsystemet#3219]
### Version number

Storefront

### Description of the bug

<img width="848" alt="example of a linktext that vanish when recives focus" src="https://github.com/user-attachments/assets/d46931c7-c2e8-4603-8d78-4e8e0d844501" />

Some links seem to have wrong colors on focus. 

This was found here: https://www.designsystemet.no/bloggen/2025/generering-av-fargar-i-fargesystemet

### Steps To Reproduce

1. Go to [this article](https://www.designsystemet.no/bloggen/2025/generering-av-fargar-i-fargesystemet)
2. Go all the way down to the bottom of the page
3. Use your keyboard to move focus to the link inside the panel
4. Admire the problem! 

### Additional Information

Also found here: 

<img width="1338" alt="Image" src="https://github.com/user-attachments/assets/9b24face-da7e-4b8f-ba33-95d1af10d53b" />
